zipkin-lens/src/translations/fr/messages.json

```
@@ -0,0 +1,36 @@
+{
+  "1 Result": "1 Résultat",
+  "Address": "Adresse",
+  "Annotations": "Annotations",
+  "Archive Trace": "Trace Archivée",
+  "Dependencies": "Dépendances",
+  "Depth": "Profondeur",
+  "Download JSON": "Télécharger JSON",
+  "Duration": "Durée",
+  "End Time": "Fin",
+  "Find a trace": "Trouver une trace",
+  "Limit": "Limite",
+  "None": "Aucun",
+  "Parent ID": "ID Parent",
+  "Please select criteria in the search bar. Then, click the search button.": "S'il vous plaît sélectionner un critère dans la barre de recherche. Et, cliquer sur le bouton de recherche",
```

I would translate the `Then` by `Puis` and also I think the verbs should be conjuged (sélectonnez VS sélectionner) ?
```suggestion
  "Please select criteria in the search bar. Then, click the search button.": "S'il vous plaît sélectionnez un critère dans la barre de recherche. Puis cliquez sur le bouton de recherche.",
```

---

Agree with you for the verbs, and replace "Et" by "Puis" 👍